src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -237,6 +239,7 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
       // First entry on resolver
       this.projectId = resolverData.project._id;
       this.myForm = this.buildFormFromData(resolverData.project);
+	  this.myForm.controls.shapeFileColour.setValue(this.shapeFileColour);
```

Comment:
Make sure `this.shapeFileColor` always exists!

Reply:
Yes it's assigned a default value and then given a value from the db if it exists.